docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,447 @@
+    { label: 'FASTQ (paired)', value: 'FASTQ-paired', },
+    { label: 'FASTQ (single)', value: 'FASTQ-single', },
+  ]
+}>
+<TabItem value="BAM-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+  	"read_group_id_in_bam": "C0HVY.2",
+  	"submitter_read_group_id": "C0HVY.2",
+  	"platform_unit": "74_8a",
+  	"is_paired_end": true,
+  	"file_r1": "test_rg3.bam",
+  	"file_r2": "test_rg4.bam",
+  	"read_length_r1": 150,
+  	"read_length_r2": 125,
+  	"insert_size": 232,
+  	"sample_barcode": null,
+  	"library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="BAM-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": "C0HVY.2", 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8a",
+    "is_paired_end": true,
+    "file_r1": "test_rg55.bam",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null,
+    "insert_size": null,
+    "sample_barcode": null,
+    "library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": true,
+    "file_r1": "test_rg23.fq",
+    "file_r2": "test_rg24.fq",
+    "read_length_r1": 150,
+    "read_length_r2": 125, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": false,
+    "file_r1": "test_rg5.fq",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null, 
+    "insert_size": null,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+### File and Data Validation Rules
+
+Sequencing data of both `BAM` and `FASTQ` type files are accepted, with the exception of interleaved FASTQ files.  Metadata payloads like the example above must follow these data validations:
+
+1. Compression of FASTQ files is **required**; both _gzip_ (suffix .fq.gz or .fastq.gz) or _bz2_ (suffix .fq.bz2 or .fastq.bz2) are supported.
+1. For both FASTQ and BAM submission, all files in the `files` section must be unique in the payload.
+1. All `read_groups` in the payload and BAM header must belong to a single sample.
+1. `platform_units` must be unique with a one-to-one relationship with `submitter_read_group_id`.
+1. The total number of `read_group` objects must match the number specified in `read_group_count`.
+1. For paired-end sequencing, both `file_r1` and `file_r2` are required.  For single-end sequencing, only `file_r1` is required (`file_r2` must be populated with `null`).
+1. For paired-end sequencing, `file_r1` and `file_r2` must not be the same file. 
```

Comment:
@rosibaj , this is NOT correct. 
For paired-end sequencing, `file_r1` and `file_r2` must not be the same file.  
---> 
For a given `submitter_read_group_id`,  if providing paired-end FASTQ files, `file_r1` and `file_r2` must not be the same file. However, for paired-end BAM files, `file_r1` and `file_r2` must be the same file.

Reply:
#403